en/micro-integrator/docs/references/connectors/salesforce-soap-connector/salesforce-soap-reference.md

```diff
@@ -0,0 +1,1121 @@
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>loginUrl</td>
+            <td>The login URL to access the Salesforce account.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>blocking</td>
+            <td>Indicates whether the connector needs to perform blocking invocations to Salesforce. (Supported in WSO2 ESB 4.9.0 and later.)</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <salesforce.init>
+        <loginUrl>{$ctx:loginUrl}</loginUrl>
+        <username>{$ctx:username}</username>
+        <password>{$ctx:password}</password>
+        <blocking>{$ctx:blocking}</blocking>
+     </salesforce.init>
+    ```
+
+    **Sample request**
+
+    ```xml
+    <salesforce.init>
+        <username>MyUsername</username>
+        <password>MyPassword</password>
+        <loginUrl>https://login.salesforce.com/services/Soap/u/42.0</loginUrl>
+        <blocking>false</blocking>
+    </salesforce.init>
+    ```
+---
+
+## Working with emails
+    
+??? note "emails"
+    The salesforcebulk.emails method creates and sends an email using Salesforce based on the properties that you specify. See the [related API documentation](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_sendemail.htm) for more information.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>sendEmail</td>
+            <td>XML representation of the email.</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+    
+    ```xml
+    <salesforce.sendEmail>
+        <sendEmail xmlns:sfdc="sfdc">{//sfdc:emailWrapper}</sendEmail>
+    </salesforce.sendEmail>
+    ```
+
+    **Sample request**
+    
+    Given below is a sample request that can be handled by the sendEmail operation.
+    
+    ```xml
+    <payloadFactory>
+       <format>
+         <sfdc:emailWrapper xmlns:sfdc="sfdc">
+         <sfdc:messages type="urn:SingleEmailMessage">
+            <sfdc:bccSender>true</sfdc:bccSender>
+            <sfdc:emailPriority>High</sfdc:emailPriority>
+            <sfdc:replyTo>123@gmail.com</sfdc:replyTo>
+            <sfdc:saveAsActivity>false</sfdc:saveAsActivity>
+            <sfdc:senderDisplayName>wso2</sfdc:senderDisplayName>
+            <sfdc:subject>test</sfdc:subject>
+            <sfdc:useSignature>false</sfdc:useSignature>
+           <sfdc:targetObjectId>00390000001PBFn</sfdc:targetObjectId>
+           <sfdc:plainTextBody>Hello, this is a holiday greeting!</sfdc:plainTextBody>    
+         </sfdc:messages>
+          </sfdc:emailWrapper>
+        </format>
+        <args/>
+    </payloadFactory>
+              
+    <salesforce.sendEmail>
+        <sendEmail xmlns:sfdc="sfdc">{//sfdc:emailWrapper}</sendEmail>
+    </salesforce.sendEmail>
+    ```
```

Comment:
Shall we add expected responses as well? It will help users know what to expect if everything works fine

Reply:
Same comment for other operations as well

Reply:
Updated.

Reply:
https://github.com/wso2/docs-ei/pull/2017